src/server/compositor/basic_screen_shooter.cpp

```diff
@@ -47,12 +47,57 @@ mc::BasicScreenShooter::Self::Self(
 {
 }
 
+auto mc::BasicScreenShooter::Self::render(
+    std::shared_ptr<mrs::WriteMappableBuffer> const& buffer,
+    geom::Rectangle const& area) -> time::Timestamp
+{
+    std::lock_guard<std::mutex> lock{mutex};
```

Comment:
You can drop the `<std::mutex>` from here; C++20 allows deduction of the template args.

Reply:
(And Alan has done a huge PR removing all the existing explicit template arguments, so it'd be nice to keep it consistent :grin: )